app/templates/gentelella/users/events/_table.html

```diff
@@ -53,15 +53,15 @@
 
                     </td>
                     <td>
-                        <ol style="padding-left:0;">
+                        <ol style="padding-left:10px;">
                             <li>{{ _("Drafts") }}: {{ drafts.count }}</li>
                             <li>{{ _("Submitted") }}: {{ event.session | length }}</li>
                             <li>{{ _("Accepted") }}: {{ accepted.count }}</li>
                             <li>{{ _("Pending") }}: {{ pending.count }}</li>
                             <li>{{ _("Rejected") }}: {{ rejected.count }}</li>
                         </ol>
                     </td>
-                    <td>{{ event.speaker | length }}</td>
+                    <td style="padding-left:30px;">{{ event.speaker | length }}</td>
```

Comment:
if no padding is given then the count for speakers was getting a bit close to session's column. that's y.